README.md

```diff
@@ -23,6 +23,7 @@ The good things about forking:
 5. Make a Pull Request (PR) from your own fork to the main repository
 6. Check whether some issues were found in the review process
 7. When ready, marvel the changes that you made to the main project after the project maintainer has merged the changes to the main repo
+8. Ready
```

Comment:
This should not be here. Please add your line at the end of the file. Also update your fork.